src/ophyd_async/epics/_backend/_aioca.py

```diff
@@ -76,16 +135,13 @@ def write_value(self, value: str):
         return value + "\0"
 
 
-class CaArrayConverter(CaConverter):
-    def get_datakey(self, source: str, value: AugmentedValue) -> DataKey:
-        return {"source": source, "dtype": "array", "shape": [len(value)]}
-
-    def value(self, value: AugmentedValue):
-        return np.array(value, copy=False)
```

Comment:
Still need the `value()` override for CaArrayConverter

Reply:
```
$ await backend.get_reading()
{'value': array([-128,  127], dtype=int8), 'timestamp': 1718287009.7993224, 'alarm_severity': 0}
```


```
$ np.array((await backend.get_reading())["value"], copy=False)
array([-128,  127], dtype=int8)
```

Seems to be being handled?

Reply:
....helps if I pick a dtype that is actually supported by AIOCA I guess, since actually it's timing out while I'm trying it

Reply:
I put some breakpoints in MonitorQueue.assert_updates and it seems to behave identically as to np.array(value, copy=False) being called on it.

Reply:
I have put it back in.